automationtestapp/src/main/java/com/microsoft/aad/automation/testapp/CustomWebViewClient.java

```diff
@@ -0,0 +1,14 @@
+package com.microsoft.aad.automation.testapp;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+public class CustomWebViewClient extends WebViewClient {
```

Comment:
Just curious why a CustomWebViewClient if we are just returning false from shouldOverrideUrl

Reply:
False indicates that the webview itself handles the redirect.  Which is fine in this case.